core/lib/sync/task.ts

```diff
@@ -182,3 +183,44 @@ export function initializeSyncTaskStorage () {
     driver: localForage[config.localForage.defaultDrivers['syncTasks']]
   }))
 }
+
+export function registerSyncTaskProcessor () {
+  const mutex = {}
+  Vue.prototype.$bus.$on('sync/PROCESS_QUEUE', data => {
+    if (typeof navigator !== 'undefined' && navigator.onLine) {
```

Comment:
we have reactive onlineHelper tor that, please use
`import { onlineHelper } from '@vue-storefront/core/helpers'`
and then
`onlineHelper.isOnline`

Reply:
OK, done!